packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -45,6 +45,10 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
       maxWidth: maxWidth!
     });
 
+    const onMouseDown = (ev: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      ev.preventDefault();
```

Comment:
Looks like this has regressed a lot of Screener scenarios...
![image](https://user-images.githubusercontent.com/4161791/77116084-88252700-69fd-11ea-830d-605425dbf156.png)
but the Screener results link itself only shows 2 that were changed in TeachingBubble this is a bit confusing... and they don't seem related to your change. I'm going to close and reopen to trigger another run.